test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -243,6 +335,40 @@ describe('CSOT spec prose tests', function () {
        *   - Expect this to fail with a timeout error.
        * 1. Verify that a `find` command was executed against the `keyvault.datakeys` collection as part of the `decrypt` call.
        */
+      it('times out due to timeoutMS', async function () {
+        const datakeyId = await clientEncryption.createDataKey('local');
+        expect(datakeyId).to.be.instanceOf(Binary);
+        expect(datakeyId.sub_type).to.equal(Binary.SUBTYPE_UUID);
+
+        const encrypted = Binary.createFromBase64(
+          'Af6ie/LRP0uoisAZthHPUs0CKzTBFIkJr8kxmOk1pV1C/6K54otT8QvNJgNTNG2CNpThhfdXaObuOMMReNlTgwapqPYCb/HJRQ1Nfma6uA3cTg==',
+          6
+        );
```

Comment:
The prose test (on the [spec](https://github.com/mongodb/specifications/tree/master/source/client-side-operations-timeout/tests#decrypt) at least) says to encrypt the value 'hello'. I assume this is the precomputed encrypted value? Any reason not to just compute it here?

Reply:
I'll add a comment describing why. Essentially, if we call decrypt right after encrypt, some data is cached and the find command that we have a failPoint on might not run - resulting in no delay and no MongoOperationTimeoutError.